docs/dev/migration.md

```diff
@@ -145,16 +145,16 @@ public class BufferingSink implements SinkFunction<Tuple2<String, Integer>>,
         bufferedElements.add(value);
         if (bufferedElements.size() == threshold) {
             for (Tuple2<String, Integer> element: bufferedElements) {
-	        // send it to the sink
-	    }
-	    bufferedElements.clear();
-	}
+            // send it to the sink
+        }
+        bufferedElements.clear();
```

Comment:
the indentation here doesn't look correct

Reply:
Tabs only indent seven spaces after the +/- markers.

Reply:
I don't know whether you agree with me or not :(

Looking at the raw code one sees that the modified section should be indented with 4 more spaces:
```
    @Override
    public void invoke(Tuple2<String, Integer> value) throws Exception {
        bufferedElements.add(value);
        if (bufferedElements.size() == threshold) {
            for (Tuple2<String, Integer> element: bufferedElements) {
            // send it to the sink
        }
        bufferedElements.clear();
    }
    }
```

Reply:
Ah, yes, @zentol you are right of course!

Reply:
I can add the 4 spaces for the above snippet 
```
diff --git a/docs/dev/migration.md b/docs/dev/migration.md
index c9a6ecfa1f..b7df4562ce 100644
--- a/docs/dev/migration.md
+++ b/docs/dev/migration.md
@@ -148,7 +148,7 @@ public class BufferingSink implements SinkFunction<Tuple2<String, Integer>>,
             // send it to the sink
         }
         bufferedElements.clear();
-    }
+        }
     }
 
     @Override
```

Reply:
BTW, should {% highlight java %} be ---java
the java code doesn't render for Linux chrome ?

Reply:
hmm, it should be `{% highlight java %}`, in particular since [FLINK-8726](https://issues.apache.org/jira/browse/FLINK-8726) popped up.